docs_src/pipelinebenchmarking.md

```diff
@@ -67,21 +73,33 @@ change the current directory to directory `benchmark_scripts`.
 
 ---
 
-### Specified number of pipelines ( Discover the the performance and system requirements for a given use case )
+### Specified number of pipelines
 
-Run benchmarking pipelines:
+One of main purposes to run benchmarking with specified number of pipelines is to discover the performance and system requirements for a given use case.
+
+An example to run benchmarking pipelines with specified number of pipelines:
 ```bash
 sudo ./benchmark.sh --pipelines <number of pipelines> --logdir <output dir>/data --init_duration 30 --duration 120 --platform <core|xeon|dgpu.x> --inputsrc <ex:4k rtsp stream with 10 objects>
 ```
 
-Get consolidated pipeline results:
+where some of configurable input parameters are:
+--logdir is to configure the benchmarking output directory
+--duration is to configure how long the benchmarking will run in number of seconds
+--init_duration is to configure how long the initial time to wait before the benchmarking to take metrics or data collection in number of seconds
```

Comment:
```suggestion
--init_duration configures how long to wait for system initialization before the benchmarking metrics or data collection begins
```

Reply:
don't have to use this wording exactly, but just cleaning it up a bit to make it more concise.